src/file.ts

```diff
@@ -2595,6 +2601,65 @@ class File extends ServiceObject<File> {
     });
   }
 
+  isPublic(): Promise<IsPublicResponse>;
+  isPublic(callback: IsPublicCallback): void;
+  /**
+   * @callback IsPublicCallback
+   * @param {?Error} err Request error, if any.
+   * @param {boolean} resp Whether file is public or not.
+   */
+  /**
+   * @typedef {array} IsPublicResponse
+   * @property {boolean} 0 Whether file is public or not.
+   */
+  /**
+   * Check whether this file is public or not.
```

Comment:
We should explain how we are doing this. It is possible the server has a hiccup and returns an error code, even though the file is public.

Reply:
Done! 
Thanks